code/tick/feed.q

```diff
@@ -31,15 +31,37 @@ randomize:{value "\\S ",string "i"$0.8*.z.p%1000000000}
 rnd:{0.01*floor 0.5+x*100}
 vol:{10+`int$x?90}
 
-/ randomize[]
-\S 235721
+randomize[]
+/ \S 235721
 
-/ =========================================================
+/ =========================================================================================
+/ generate weights to stop even distribution of counts and sizes
+
+weight:0.3 0.6 1.8 1.1 1.2 1.6 0.9 0.4 0.7 1.4
```

Comment:
Should these be randomised?

Reply:
Do you mean each sym should have one weighting for everything? It would mean the bid and ask volumes would be the same for each sym

Reply:
No I'm asking should this list of numbers be randomised instead of hardcoded? And should all this logic be wrapped up into a function in case there isn't 10 syms so the number can be varied easily?

Reply:
think something like this might work?

```suggestion
weight:0.1*neg[cnt]?2*cnt
```